### ISSUE TITLE
Regression : Can't deploy from a "not-master" branch 

### ISSUE THREAD

**Issue by Korbik** [CleverCloud/clever-tools#187]
I had this error with the new version on MacOs. 

```Pushing source code to Clever Cloud.
/snapshot/workspace/node_modules/nodegit-promise/lib/done.js:10
      throw err;
      ^

TypeError: repository.getCurrentBranch is not a function
    at /snapshot/workspace/src/models/git.js:71:43
    at /snapshot/workspace/node_modules/baconjs/dist/Bacon.js:2554:27
    at spawn (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:2124:34)
    at Object.sink (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:2173:20)
    at Dispatcher.pushToSubscriptions (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1223:23)
    at Dispatcher.pushIt (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1245:10)
    at Object.inTransaction (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:430:16)
    at Dispatcher.push (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1214:24)
    at Dispatcher.handleEvent (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1267:17)
    at _.bind (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:242:17)
    at Object.sink (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1740:16)
    at PropertyDispatcher.Dispatcher.pushToSubscriptions (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1223:23)
    at PropertyDispatcher.Dispatcher.pushIt (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1245:10)
    at Object.inTransaction (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:430:16)
    at PropertyDispatcher.Dispatcher.push (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1214:24)
    at PropertyDispatcher.push (/snapshot/workspace/node_modules/baconjs/dist/Bacon.js:1655:38)
```

I have made a complete uninstall of npm and clever-tools

**Comment by hsablonniere** [comment]
Hello @Korbik :smile:

From your issue title, I get that you tried to deploy a branch which is not named `master`.

* Can you tell me exactly which command you ran (including options)?
* How did you install the Clever Tools ?
* Which version are you using? (`clever --version` will tell you)

**Comment by hsablonniere** [comment]
I think I found the source of the problem. I released a new `0.10.1` version. Can you check if you still have the problem?

**Comment by Korbik** [comment]
😄  Ok, I was a little bit in an hurry. The version is 0.10.0. I had used brew install (after cleaning all the installations). The version 0.10.1 has fixed the issue.
